lib/awestruct/extensions/disqus.rb

```diff
@@ -46,7 +46,7 @@ def disqus_comments_count()
             var disqus_shortname = '#{site.disqus}';
             (function () {
               var s = document.createElement('script'); s.async = true;
-              s.src = "http://disqus.com/forums/#{site.disqus}/count.js";
+              s.src = '//' + disqus_shortname + '.disqus.com/count.js';
```

Comment:
Ah :) my bad, I wasn't paying attention. Didn't notice this was JS.

On Wednesday, May 20, 2015, Emmanuel Bernard <notifications@github.com>
wrote:

> In lib/awestruct/extensions/disqus.rb
> <https://github.com/awestruct/awestruct/pull/474#discussion_r30752549>:
>
> > @@ -46,7 +46,7 @@ def disqus_comments_count()
> >              var disqus_shortname = '#{site.disqus}';
> >              (function () {
> >                var s = document.createElement('script'); s.async = true;
> > -              s.src = "http://disqus.com/forums/#{site.disqus}/count.js";
> > +              s.src = '//' + disqus_shortname + '.disqus.com/count.js';
>
> Same here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/awestruct/awestruct/pull/474/files#r30752549>.
>


-- 
Sent from Gmail Mobile